app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -529,6 +532,18 @@ export default {
               this.addPatchToHistory();
             }
           }
+          this.canSave = true;
+          modified?.components?.map((comp) => {
+            if (comp.key === 'form') {
```

Comment:
My concern here is that it's a breaking change. A form that was fine before this PR will produce an error when edited and saved after this PR, even if there are no changes to the form.

Maybe this isn't a big deal - how hard would it be to identify how many forms already in production use "form" as one of the field names? Maybe we could work with those form owners to fix their forms before the change.

Reply:
@WalterMoar there are only 2 published forms which have `form` as a field name, should we just notify them?